.travis.yml

```diff
@@ -22,6 +25,13 @@ jobs:
       julia: 1.0
       os: linux
       script:
+        # html
         - julia --project=docs/ -e 'using Pkg; Pkg.develop(PackageSpec(path=pwd())); Pkg.instantiate()'
         - julia --project=docs/ docs/make.jl
-      after_success: skip
+        # pdf
+        # - julia --project=docs/pdf/ -e 'using Pkg; Pkg.develop(PackageSpec(path=pwd())); Pkg.instantiate()'
+        - julia --project=docs/pdf/ -e 'using Pkg; Pkg.instantiate()'
+        - julia --project=docs/pdf/ docs/pdf/make.jl
+      after_success:
+        - julia --project=coverage/ -e 'using Pkg; Pkg.instantiate()'
```

Comment:
Not sure this is a good idea. Maybe just build the pdf during regular testing?

Reply:
I was thinking that we could have a separate build stage for this.

Reply:
If we can, let's do it. If not, let's leave LaTeX uncovered for now. Although, no deployment at the moment, right? In which case, we could indeed also just build the PDF during testing (as long as the build time does not blow up too much).